libs/designsystem/src/lib/components/reorder-list/reorder-list.component.html

```diff
@@ -1,7 +1,7 @@
 <ion-backdrop *ngIf="reorderActive" [tappable]="false" [visible]="true" [stopPropagation]="false">
 </ion-backdrop>
 <ion-reorder-group (ionItemReorder)="doReorder($event)" disabled="false">
-  <div *ngFor="let reorderItem of items">
```

Comment:
Please implement track by function in *ngFor due to performance and possibility to change items of the list without rendering the whole list again. 
https://medium.com/better-programming/improving-angular-ngfor-performance-through-trackby-ae4cf943b878